src/Screens/Cart/index.tsx

```diff
@@ -1,14 +1,27 @@
-import { NavigationContainer } from "@react-navigation/native";
-import React from "react";
+import React, {useState,useEffect} from "react";
 import {View,Text} from 'react-native'
 import styles from "./styles";
 import {Header,CartCard,Button} from "components"
 
+import { connect } from "react-redux";
 
-
-const Cart = ({navigation,route}) =>{
+const Cart = ({navigation,route,cart}) =>{
     const item = route.params
-    
+    const [totalPrice,setTotalPrice] = useState(0);
+    const [totalItem,setTotalItem] = useState(0);
+
+    useEffect(()=>{
```

Comment:
every time cart screen is opened the items added into cart in previous run should display ,
do the UI changes for item card